druid/src/contexts.rs

```diff
@@ -87,12 +76,10 @@ pub struct UpdateCtx<'a> {
 /// As of now, the main service provided is access to a factory for
 /// creating text layout objects, which are likely to be useful
 /// during widget layout.
-pub struct LayoutCtx<'a, 'b: 'a> {
-    pub(crate) command_queue: &'a mut CommandQueue,
+pub struct LayoutCtx<'a, 'b: 'a, 'c> {
+    pub(crate) root_state: &'a mut RootState<'c>,
```

Comment:
This might be a bit annoying to change but it would be nice if all `'b`s refer to the `RootState`s lifetime.

Reply:
yea that would be nice but doesn't work unfortunately. :(

Reply:
How so? The only 'issue' I see is `PainCtx` as it does not have `RootState` :thinking:

Reply:
oh sorry I thought you meant that we reuse `'b` here for both inner lifetimes.